### ISSUE TITLE
EFI Error (0x0100000000000082)

### ISSUE THREAD

**Issue by LLLLLMMMMMAAAOOO** [terminatorul/NvStrapsReBar#15]
**System**
 - Motherboard: B460M-A-PRO
 - BIOS Version: 7C88v19
 - GPU: GTX 1650 SUPER
 - [ ] CSM is turned off. **Make sure to confirm this in the BIOS and not with GPU-Z or similar since it can be inaccurate**
 - [ ] 4G decoding is enabled. **Make sure to confirm this in the BIOS and not with GPU-Z or similar since it can be inaccurate**
 - [ ] UEFIPatch is applied (see [Using UEFIPatch](https://github.com/xCuri0/ReBarUEFI/wiki/Using-UEFIPatch) for more information). On some motherboards [DSDT Patching](https://github.com/xCuri0/ReBarUEFI/wiki/DSDT-Patching) is also needed
 - [ ] I have read [Common issues (and fixes)](https://github.com/xCuri0/ReBarUEFI/wiki/Common-issues-(and-fixes))

I successfully enabled rebar via checking on GPU-Z, setting a specific amount of BAR Size in NvStrapsReBar also works. Also checked my GPU under device manager and it shows under Large Memory just to verify if ReBar is actually enabled.

ISSUE: I'm having the EFI DXE driver status: EFI Error (0x0100000000000082) on NvStrapsReBar.exe

![image](https://github.com/terminatorul/NvStrapsReBar/assets/91077891/0ba84ba2-7f3d-48db-aa2a-13e41fbf1627)

**Comment by terminatorul** [comment]
Can you build and re-flash the latest version from the `bleed` branch ?

**Comment by LLLLLMMMMMAAAOOO** [comment]
I just renamed the .rom file created with NvStrapsReBar.ffs inserted then flash, cause DSDT on my mobo is not required + UEFIPatch no work just parse error.

**Comment by LLLLLMMMMMAAAOOO** [comment]
UPDATE:

![image](https://github.com/terminatorul/NvStrapsReBar/assets/91077891/fd9be8e4-b2c5-4b77-997e-cc783a7f2c82)

Thank you!

**Comment by terminatorul** [comment]
Could you also post your system information in issue https://github.com/terminatorul/NvStrapsReBar/issues/1, to help other people with the same hardware combination ?

**Comment by LLLLLMMMMMAAAOOO** [comment]
done posting.

Is it okay to leave Target PCI BAR size at default instead of Selected GPUs Only?

![image](https://github.com/terminatorul/NvStrapsReBar/assets/91077891/8dd516e1-9a19-438e-a562-2a674a640b9f)

**Comment by terminatorul** [comment]
Yes ! This is the recommended setting for all systems (motherboards) that support it
